x/subspaces/keeper/groups.go

```diff
@@ -0,0 +1,113 @@
+package keeper
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+
+	"github.com/desmos-labs/desmos/v2/x/subspaces/types"
+)
+
+// SetGroupID sets the new group id for the specific subspace to the store
```

Comment:
I would name it to `SetNextGroupID` since `SetGroupID` makes me think it the current group id.
And should we make it private method to prevent other modules change it?

Reply:
`SetGroupID` is used to match `GetGroupID`, which returns the group id to be used when creating a group id. I think the documentation is clear enough, and it's better to match the name of the getter instead (like it is right now). 

I think it's fine keeping it public since we need it inside tests as well.